lexer/tokenizer.go

```diff
@@ -49,9 +48,7 @@ func belongsTogether(current []rune, next rune) bool {
 		switch current[0] {
 		case '+':
 			return next == '+'
-		case '!':
-			fallthrough
-		case '=':
+		case '!', '=':
```

Comment:
Ah great, that ist what I was searching for, but it wasn't used in any of the examples. So I thought it's not possible in Go :D